tests/styles.spec.js

```diff
@@ -0,0 +1,43 @@
+const assert = require('assert');
+const { execSync } = require('child_process');
+const { mkdirSync, cpSync, readFileSync } = require('fs');
```

Comment:
Looks like I also added `cpSync` here where we didn’t need it.
```suggestion
const { mkdirSync, readFileSync } = require('fs');
```

Sorry about that!